.github/workflows/flatpak-build.yml

```diff
@@ -0,0 +1,19 @@
+on:
+  push:
+    branches: [main]
+  pull_request:
+name: CI
+jobs:
+  flatpak:
+    name: "Flatpak"
+    runs-on: ubuntu-latest
+    container:
+      image: bilelmoussaoui/flatpak-github-actions:gnome-44
```

Comment:
I had problems using this - we should probably use https://github.com/flathub-infra/flatpak-github-actions

Reply:
Should also use freedesktop instead of gnome